cmd/virt-handler/virt-handler.go

```diff
@@ -50,26 +50,33 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-handler/virtwrap/isolation"
 )
 
-func main() {
-	logging.InitializeLogging("virt-handler")
-	libvirt.EventRegisterDefaultImpl()
-	libvirtUri := flag.String("libvirt-uri", "qemu:///system", "Libvirt connection string.")
-	listen := flag.String("listen", "0.0.0.0", "Address where to listen on")
-	port := flag.Int("port", 8185, "Port to listen on")
-	host := flag.String("hostname-override", "", "Kubernetes Pod to monitor for changes")
-	socketDir := flag.String("socket-dir", "/var/run/kubevirt", "Directory where to look for sockets for cgroup detection")
-	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
-	pflag.Parse()
+type VirtHandlerApp struct {
+	Service      *service.Service
```

Comment:
should all of this be lower case since it isn't exported outside of this package?

Reply:
I think it is fine. Right now it is not, but it is something which is intended to be used everywhere in virt-handler code to provide configs. Like for virt-controller, this app will then have to go to /pkg/virt-handler. So works for me.